lib/recurly/frame.js

```diff
@@ -232,6 +240,8 @@ export class Frame extends Emitter {
   createIFrame () {
     const { container, url } = this;
     const iframe = document.createElement('iframe');
+    const dimensions = this.constructor.DIMENSIONS;
+    const windowSize = this.recurly.config.challengeWindowSize;
 
```

Comment:
We can add a simple validation for this:
```suggestion

    const selectedWindowSize = dimensions[windowSize];
    if (!selectedWindowSize) {
      const validWindowSizes = Object.keys(dimensions).join(', ');
      throw new Error(`Invalid challengeWindowSize. Expected any of ${validWindowSizes}, got ${windowSize}`);
    }

```
Also, the newly created `selectedWindowSize` variable could be used directly when setting the attributes in line 255.

Reply:
I like it. More validation the better I'd say